app/updater.js

```diff
@@ -85,6 +86,35 @@ var scheduleUpdates = () => {
     }, appConfig.updates.appUpdateCheckFrequency)
   }
 
+  // Linux doesn't have an auto-update mechanism.
+  // Instead, deliver a periodic nag linking to instructions
+  if (isLinux() && appConfig.updates.deprecatedNagFrequency) {
+    electron.ipcMain.on(messages.NOTIFICATION_RESPONSE, (e, message, buttonIndex, persist) => {
+      appActions.hideNotification(message)
```

Comment:
i thought we decided to just make the notification unhideable on linux? fine with either way, just wondering if product spec has changed.

Reply:
Whoops- my bad on that. I'll fix that up 😄 So much for having a legit use for a `nagFrequency` variable 😂